scte35/state.go

```diff
@@ -162,13 +163,6 @@ func (s *state) ProcessDescriptor(desc SegmentationDescriptor) ([]SegmentationDe
 			err = gots.ErrSCTE35MissingOut
 			break
 		}
-		for _, d := range closed {
```

Comment:
this closing logic doesn't seem to be what we need in SCTE35-2020. Removing it for that reason.